docs/source/getting-started/installation.rst

```diff
@@ -97,15 +96,25 @@ Google Cloud Platform (GCP)
   $ gcloud auth application-default login
 
 
-If running :code:`conda install -c conda-forge google-cloud-sdk` produces the error *"RemoveError: 'requests' is a dependency of conda and cannot be removed from conda's operating environment"*, try :code:`conda update --force conda` first and rerun the command.
+Note: If you encounter "*RemoveError: 'requests' is a dependency of conda and
+cannot be removed from conda's operating environment*" when running :code:`conda
+install -c conda-forge google-cloud-sdk`, try :code:`conda update --force conda`
+first and rerun the command.
 
-Note: if you encounter *Authorization Error (Error 400: invalid_request)* with the url generated by :code:`gcloud auth login`, try installing the latest version of the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_ (e.g., with :code:`conda install -c conda-forge google-cloud-sdk`) on **your local machine** (which opened the browser) and rerun the command.
+Note: If you encounter "*Authorization Error (Error 400: invalid_request)*" with
+the url generated by :code:`gcloud auth login`, try installing the latest
+version of the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_
+(e.g., with :code:`conda install -c conda-forge google-cloud-sdk`) on your local
+machine (which opened the browser) and rerun the command.
 
 .. tip::
 
-  If you are using multiple GCP projects, list all the projects by :code:`gcloud projects list` and activate one by :code:`gcloud config set project <PROJECT_ID>` (See `GCP docs <https://cloud.google.com/sdk/gcloud/reference/config/set>`_).
+  If you are using multiple GCP projects, list all the projects by :code:`gcloud projects list` and activate one by :code:`gcloud config set project <PROJECT_ID>` (see `GCP docs <https://cloud.google.com/sdk/gcloud/reference/config/set>`_).
 
-To use service account to access GCP for SkyPilot, see :ref:`here<gcp-service-account>` for instructions.
+**Optional**: To use service accounts to access GCP for SkyPilot, see
+:ref:`here<gcp-service-account>` for instructions. A long-lived service account
+on your local machine can avoid the periodic
+"*google.auth.exceptions.RefreshError: Reauthentication is needed.*" error.
```

Comment:
Should we wrap these into a [note box](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes) and clearly enumerate the error and suggested fix in bullet points/table? Otherwise these notes are becoming hard to read and easy to miss.

E.g. something like
```
.. note::

  Here some commonly encountered errors and their fix:
    * "*RemoveError: 'requests' is a dependency of conda and
cannot be removed from conda's operating environment*" when running :code:`conda
install -c conda-forge google-cloud-sdk` - run :code:`conda update --force conda`
first and rerun the command.
    * "*Authorization Error (Error 400: invalid_request)*" with
the url generated by :code:`gcloud auth login` - install the latest
version of the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_
(e.g., with :code:`conda install -c conda-forge google-cloud-sdk`) on your local
machine (which opened the browser) and rerun the command.
    * "*google.auth.exceptions.RefreshError: Reauthentication is needed.*" - use a long-lived service account
on your local machine. See :ref:`here<gcp-service-account>` for instructions.
```
We can additionally include a separate mention of GCP service account support.

Reply:
Now put the errors in a dropdown, which I think makes the installation less daunting. Kept the first two bullets only as the last bullet is technically not encountered in installation.